README.md

```diff
@@ -35,7 +35,12 @@ package in 'developer mode' using `pip`, in a virtual environment. You also need
 ```sh
 $ pip install -e .
 ```
+## Prerequisites
+You'll need:
+- pandoc: http://pandoc.org/installing.html
```

Comment:
Not at the moment. We used it at the beginning to convert privacy policies to github markdown but disabled this feature temporarily. We might do it again when we have a better cleaning process for the policies.